### DISCUSSION TITLE
[Unleash proxy] Fetching all feature flags from project

### DISCUSSION THREAD

**Discussion by wuja-feely** [Unleash/unleash#7588] [Q&A] [answered]
Hello,

I am working on creating a feature flags handler that sets the value to true if a specific feature flag (ff) is not defined in the GitLab implementation of Unleash. Is there a standard method for achieving this?

Using the `/proxy/all` endpoint returns both enabled and disabled flags only for a specific environment. The only solution I can think of is fetching all feature flags for the entire project (along with their values based on strategies) and then checking if a specific feature flag exists.

Has anyone encountered this issue before?

Thank you.

**Comment by chriswk** [comment, marked as answer]
Hi. The Unleash Proxy does not support multiple environments. If you want multiple environments you'll have to spin up a proxy per environment
